tests/download_test_data.py

```diff
@@ -0,0 +1,26 @@
+import requests
+
+def download_file(url, local_filename):
```

Comment:
We have code to download datashare files around different repositories. 
e.g https://github.com/MannLabs/alphadia/blob/main/alphadia/testing.py

maybe we can move them to alphaBase or some other package.

Reply:
I think this would be worth a dedicated repo? let's gather more candidates in the ticket I created: https://github.com/orgs/MannLabs/projects/12/views/1?pane=issue&itemId=63254864

for now, I could live with this duplication (not biased by, but yet aware of the fact that I added the exact same duplication less than 24h ago :-) )

Reply:
Yes, I saw our code is almost the same for downloading data, we should create a repo for GitHub utils. Actually, github_template is a good idea, similar to AlphaTemplate